static/usage/modal/inline/basic/angular/app_component_html.md

```diff
@@ -0,0 +1,30 @@
+```html
+<ion-app>
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>Inline Modal</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content class="ion-padding">
+    <ion-button id="open-modal" expand="block">Open</ion-button>
+    <p>{{ message }}</p>
+    <ion-modal trigger="open-modal" (didDismiss)="onDidDismiss($event)">
```

Comment:
Optional: I wonder if it would look nicer if we used willDismiss. The benefit is that the name will already have been rendered by the time the modal animation reveals the underlying page. (Unless the data is only passed back on didDismiss?)